pom.xml

```diff
@@ -20,7 +20,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>io.axonif</groupId>
   <artifactId>queuebacca</artifactId>
-  <version>0.2.2</version>
+  <version>0.2.3</version>
```

Comment:
Don't worry about upgrading the version, I usually do it in a commit prior to putting it into the master branch.